public/keymaps/d/durgod_k320_default.json

```diff
@@ -0,0 +1,24 @@
+{
+  "keyboard": "durgod/k320",
+  "keymap": "default",
+  "commit": "537b7614b9be39025379291880b549e4fa4d826d",
+  "layout": "LAYOUT_tkl_ansi",
+  "layers": [
+    [
+        "KC_ESC",   "KC_F1",    "KC_F2",    "KC_F3",    "KC_F4",    "KC_F5",    "KC_F6",    "KC_F7",    "KC_F8",    "KC_F9",    "KC_F10",   "KC_F11",   "KC_F12",               "KC_PSCR",  "KC_SLCK",  "KC_PAUS",
+        "KC_GRV",   "KC_1",     "KC_2",     "KC_3",     "KC_4",     "KC_5",     "KC_6",     "KC_7",     "KC_8",     "KC_9",     "KC_0",     "KC_MINS",  "KC_EQL",   "KC_BSPC",  "KC_INS",   "KC_HOME",  "KC_PGUP",
+        "KC_TAB",   "KC_Q",     "KC_W",     "KC_E",     "KC_R",     "KC_T",     "KC_Y",     "KC_U",     "KC_I",     "KC_O",     "KC_P",     "KC_LBRC",  "KC_RBRC",  "KC_BSLS",  "KC_DEL",   "KC_END",   "KC_PGDN",
+        "KC_CAPS",  "KC_A",     "KC_S",     "KC_D",     "KC_F",     "KC_G",     "KC_H",     "KC_J",     "KC_K",     "KC_L",     "KC_SCLN",  "KC_QUOT",  "KC_ENT",
+        "KC_LSFT",  "KC_Z",     "KC_X",     "KC_C",     "KC_V",     "KC_B",     "KC_N",     "KC_M",     "KC_COMM",  "KC_DOT",   "KC_SLSH",  "KC_RSFT",                          "KC_UP",
+        "KC_LCTL",  "KC_LGUI",  "KC_LALT",                                "KC_SPC",                                 "KC_RALT",  "MO(1)",    "KC_APP",   "KC_RCTL",  "KC_LEFT",  "KC_DOWN",  "KC_RGHT"
```

Comment:
Other than a bit of whitespace to move the KC_LEFT stuff over, I don't see a problem with these keys on either of these layers.  :+1:  Thanks a bunch for doing this.  I've been wanting to try the configurator out myself.